internal/kymacustomresource/kyma_cr_updater.go

```diff
@@ -21,33 +21,36 @@ const (
 )
 
 type Updater struct {
-	k8sClient     dynamic.Interface
-	queue         syncqueues.MultiConsumerPriorityQueue
-	kymaGVR       schema.GroupVersionResource
-	sleepDuration time.Duration
-	labelKey      string
-	ctx           context.Context
-	logger        *slog.Logger
+	k8sClient                 dynamic.Interface
+	queue                     syncqueues.MultiConsumerPriorityQueue
+	kymaGVR                   schema.GroupVersionResource
+	sleepDuration             time.Duration
+	betaEnabledLabelKey       string
+	usedForProductionLabelKey string
```

Comment:
Do we expect changes to label keys? There are consts which seem to be a good replacement for struct values. If you would like to keep Updater generic, you can use map[string]string to hold labels for CR.

Reply:
Consts: https://github.com/kyma-project/kyma-environment-broker/pull/1870/files#diff-ce603fefa5e9e97af281cee331b000cc5e5dee934ab38b6022cc977194a12bd1R28-R29